pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -1140,6 +1136,16 @@ class Build {
             data.binary_type = type
             data.sha256 = hash
 
+            // generate sha256.txt
```

Comment:
this block is similar to the logic in line 1126, except  skip sbom*
why not just reuse $hash and $file